### ISSUE TITLE
xen should destroy instance on reboot

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#221]
Originally reported of Google Code with ID 165.

```
When a xen instance is rebooted, currently xen restarts it immediately with the original config. This means that restarting the instance to pick up a new kernel, or different parameters, doesn't work how users expect. A particular pain-point is that if special parameters (kernel/initrd/args) are used to start an install, when the installer finishes and reboots the instance, the install will just re-start.
```
Originally added on 2011-06-06 13:35:01 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was implemented and will be in 2.5.
```
Originally added on 2011-08-05 13:55:05 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.5

**Comment by Ganeti-Issues-Migrator** [comment]
```
Migrate old issues from 'fixed' to 'released'.
```
Originally added on 2012-10-22 09:16:17 +0000 UTC.

Changed State: Released
